index.md

```diff
@@ -67,7 +67,7 @@ Not sure where to start? Check out [our resources](#resources).
 _For those thinking: “We don’t have anything yet, where do we start?”_
 
 **Templates and Example Policies:**
-* [Resources to Help Address Sexual Harassment in Venture Ecosystem](https://nvca.org/pressreleases/nvca-unveils-resources-help-address-sexual-harassment-venture-ecosystem/) (NVCA), which includes (1) sample H.R. policies for addressing harassment & discrimination; (2) sample H.R. best practices for addressing harassment & discrimination; and (3) a sample Code of Conduct
+* NVCA recently published [resources to help address sexual harassment in the venture ecosystem](https://nvca.org/pressreleases/nvca-unveils-resources-help-address-sexual-harassment-venture-ecosystem/), which includes (1) sample H.R. policies for addressing harassment & discrimination; (2) sample H.R. best practices for addressing harassment & discrimination; and (3) a sample Code of Conduct
 * [Project Include](http://projectinclude.org/), a community for building meaningful, enduring diversity and inclusion into tech companies, has a number of templates available to use
```

Comment:
suggested language change: "a non-profit focused on building meaningful, enduring diversity and inclusion within tech startups and as well as their broader ecosystem" (or shorten it because the focus here is on the templates not on PI)

Reply:
Missed this one. Adding it in.